src/plugins/imagery/components/ImageControls.vue

```diff
@@ -0,0 +1,272 @@
+                title="Remove zoom and pan"
+                @click="handleResetImage"
+        ></button>
+
+        <span class="c-image-controls__zoom-factor">x{{ formattedZoomFactor }}</span>
+    </div>
+    <div class="c-image-controls__control c-image-controls__brightness-contrast">
+        <span
+            class="c-image-controls__sliders"
+            draggable="true"
+            @dragstart="startDrag"
+        >
+            <div class="c-image-controls__input icon-brightness">
+                <input
+                    v-model="filters.contrast"
+                    type="range"
+                    min="0"
+                    max="500"
+                    @change="notifyFiltersChanged"
+                >
+            </div>
+            <div class="c-image-controls__input icon-contrast">
+                <input
+                    v-model="filters.brightness"
+                    type="range"
+                    min="0"
+                    max="500"
+                    @change="notifyFiltersChanged"
+                >
+            </div>
+        </span>
+        <span class="t-reset-btn-holder c-imagery__lc__reset-btn c-image-controls__btn-reset">
+            <button class="c-icon-link icon-reset t-btn-reset"
+                    @click="handleResetFilters"
+            ></button>
+        </span>
+    </div>
+</div>
+</template>
+
+<script>
+import _ from 'lodash';
+
+const DEFAULT_FILTER_VALUES = {
+    brightness: '100',
+    contrast: '100'
+};
+
+const ZOOM_LIMITS_MAX_DEFAULT = 20;
+const ZOOM_LIMITS_MIN_DEFAULT = 1;
+const ZOOM_STEP = 1;
+
+export default {
+    inject: ['openmct', 'domainObject'],
+    props: {
+        zoomFactor: {
+            type: Number,
+            required: true
+        },
+        imageUrl: String
+    },
+    data() {
+        return {
+            altPressed: false,
+            shiftPressed: false,
+            metaPressed: false,
+            panZoomLocked: false,
+            wheelZooming: false,
+            filters: {
+                brightness: 100,
+                contrast: 100
+            }
+        };
+    },
+    computed: {
+        formattedZoomFactor() {
+            return Number.parseFloat(this.zoomFactor).toPrecision(2);
+        },
+        cursorStates() {
+            const isPannable = this.altPressed && this.zoomFactor > 1;
+            const showCursorZoomIn = this.metaPressed && !this.shiftPressed;
+            const showCursorZoomOut = this.metaPressed && this.shiftPressed;
+            const modifierKeyPressed = Boolean(this.metaPressed || this.shiftPressed || this.altPressed);
+
+            return {
+                isPannable,
+                showCursorZoomIn,
+                showCursorZoomOut,
+                modifierKeyPressed
+            };
+        }
+    },
+    watch: {
+        imageUrl(newUrl, oldUrl) {
+            // reset image pan/zoom if newUrl only if not locked
+            if (newUrl && !this.panZoomLocked) {
```

Comment:
@charlesh88 : one edge I was wondering about is resolution change in images (not sure if we need to address that)

Reply:
Resolved during standup with Charles.